index.html

```diff
@@ -1412,6 +1412,123 @@ <h3>Biometrics</h3>
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  
+        Avoidance of excess energy use that does not further other equally important 
+        goals such as human rights or personal privacy, particularly when that energy 
+        is sourced from non-sustainable energy creation methods, should be a best 
+        practice that serves the interest of all.
+      </p>
+      <p>
+        Direct comparison of each factor, and the balances between them, should be
+        considered when evaluating particular methods or approaches. Notably, the
+        ability to misuse or control DIDs that could impede privacy might be
+        found to be inversely correlated with additional energy consumption.
+      </p>
+      <p>
+        The guiding principle that 
+        <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+        web must be an environmentally sustainable platform</a> should be followed.
+      </p>
+
+      <p class="advisement">
+        Utilizing or authoring DID methods that require unsustainably-sourced 
+        energy as part of their technical implementation or utilization should
+        be very carefully considered in balance against alternative approaches.
+      </p>
+
+      <p>
+        When selecting a DID method, the method that minimizes energy consumption 
+        while meeting privacy, interoperability, and other technical requirements
+        should generally be chosen.
+      </p>
+
+      <p class="advisement">
+        It is strongly recommended that authors of DID methods provide an assessment
+        of energy usage and impact of their DID method, preferably performed by
+        an independent third party.
+      </p>
+
+      <p>
+        The energy usage and other environmental impacts of a DID-based solution
+        should also be compared to existing approaches, which are often paper based
+        and/or require shipment of legacy certificates via air or other methods, and
+        thus can have extremely high environmental impacts. These should be factored
+        in when performing an assessment of energy usage and environmental impacts
+        of potential replacement DID-based solutions.
+      </p>
```

Comment:
This section should be drastically simplified because it (still) reads like an attack on proof-of-work.  That was clearly intended, but is not appropriate for the following reasons:

* there is technically no other way to provide the decentralization functionality that proof-of-work ledgers provide to users who need it most;

* Bitcoin-anchored DID methods do not increase the hashrate when a user joins on, and they would not decrease the hashrate if they were magically shut off;

* the cost the end user pays to use the DID method is already a good indicator of the energy-inclusive expense of creating their DID, so there's no need to ask DID method editors to attempt an estimate when more accurate data is impossible to gather;

* <a href="https://medium.com/@nic__carter/noahbjectivity-on-bitcoin-mining-2052226310cb">Bitcoin can run exclusively on nonrival energy</a>;

* CBECI's dramatic Bitcoin mining energy use estimates (as linked earlier in this thread) rely on stale inefficient hardware assumptions which make the numbers look bigger; and

* there is a lot of <a href="https://medium.com/@nic__carter/on-bitcoin-the-gray-lady-embraces-climate-lysenkoism-a2d31e465ec0">other misinformation that takes time to explain</a> out there about the whole energy argument, and we need less scaremongering.

I hope that explains why this undue focus on proof-of-work energy usage is inappropriate from a technical perspective.  My edits above allow the W3C TAG Ethical Web Principles their say, which is enough.

For more information, in addition to the articles linked above, Andreas Antonopoulos recently made <a href="https://www.youtube.com/watch?v=xoEId1UEsG4">an 8-minute video about energy regulation and carbon taxation</a> as it relates to Bitcoin mining.

Reply:
As an editor (and the author of this original PR) I like the additional language you have added.  That language is very clean.  The key thing from this PR is is to show that regardless of personal feelings that there is a lot of balance that needs to be carefully considered, and that we clearly show to voting parties that we have considered possible arguments both for and against POW.  There is a clear balance between energy usage, privacy, and application of the tech (e.g. some problems are worth throwing high amounts of resources) that the developer should consider when selecting or authoring a did method, including overall purpose of that method.  Being that this is non-normative note, calling out specific details of things to consider is not, nor should be considered harmful.  I agree, that in many (if not most cases) energy use criticisms of certain ledgers are unfounded, especially as we see a move towards location near and sourcing of energy from renewable sources, as well as a general trend (driven by economics) to source power from "overflow" or excess generation as required to keep the grid stable.  These are good trends.  

Selection of an approach that has higher energy consumption, but that sources that energy from renewables, and solves a problem related to forced labor with better privacy and individual control than any other technical approach, would be highly justified and defensible, as opposed to an approach that gives up privacy or individual control.  Laying that thought process out is helpful. 

The reality is, that as an editor on this guide, I need to consider the arguments being raised against DIDs as a whole, regardless of my feelings or thoughts on the matter, and right now several prominent groups/individuals, primarily those who have not been engaged with DIDs from either a spec or implementation perspective, are publicly raising concerns regarding possible negative ESG impacts from the use of DIDs.  

While within the community these conversations have been had, and the trade offs are well understood, outside the community headlines grab attention, and it is up to us to demonstrate to the outside world that "we have considered these trade offs, and you should too".  As an editor, I have to consider the public "attack on proof-of-work", even though I feel that it has inherent fundamental flaws, especially because it looks only at the "Environmental" aspect of ESG, and then, only narrowly, not looking at the total energy balance and benefits, especially on the "Social" aspect of ESG.  This is very clear when we look at paper or plastic based systems for people and things identification systems in place which have huge overheads from an energy and environmental impact standpoint, especially in the supply chain.  If we can document some of these trade offs an concerns, and clearly show that as a community we have considered these items, and encourage developers to do so as well, then, while certain objections to DIDs based on ESG concerns may not be retracted, they certainly can be shown not yo be a a reason to reject the spec and approach as a whole.   

In general there is a balance to be had. If this were the spec itself, i would strongly object to adding any of this kind of language.  As guidance to a developer however, thinking through the full implications of an approach is a good thing, as too often we see the adoption of cool new tech for its own sake, and then pay for the consequences as a society later.

Reply:
> As an editor (and the author of this original PR) I like the additional language you have added.

LOL, "stop hitting yourself!"

I do not consent to the use of my name on the patch as you have applied it, because I do not consent to leaving 7.2 as is.

I proposed to remove the inappropriate language in section 7.2, for the reasons I've given, and the words I offered were *in place of* it.  My patch should be applied as it was offered.

> [...] and that we clearly show to voting parties that we have considered possible arguments both for and against POW.

That's what this Github record is for.  The plainly negative scaremongering going on in section 7.2 does not show such consideration.  The way to remove the scaremongering is to apply my patch as intended.

> [...] and right now several prominent groups/individuals, primarily those who have not been engaged with DIDs from either a spec or implementation perspective, are publicly raising concerns regarding possible negative ESG impacts from the use of DIDs.

"Concerns regarding possible" is not a specific complaint that can be resolved.  The specification process works best when technical concerns are brought to be judged on the technical merits.  That is what is going on in this proposed pull request, with specific proposed words.  You are responsible for words that you propose until the unnamed parties show up and take responsibility for words that they propose.  Either way, the technical merits should prevail.

> [...] If we can document some of these trade offs an concerns, and clearly show that as a community we have considered these items, and encourage developers to do so as well, then, while certain objections to DIDs based on ESG concerns may not be retracted, they certainly can be shown not yo be a a reason to reject the spec and approach as a whole.

Our discussions show what we have considered.  We do not need to scare developers using vague derogatory warnings that don't stand on the technical merits.

> In general there is a balance to be had.  If this were the spec itself, i would strongly object to adding any of this kind of language.  As guidance to a developer however, thinking through the full implications of an approach is a good thing, as too often we see the adoption of cool new tech for its own sake, and then pay for the consequences as a society later.

My patch, offering a Section 7.2 pointing at the TAG principles, offers balance.

The existing Section 7.2's vague derogatory warnings about energy usage are not okay, because as I've already shown in this thread, they promote an unbalanced and ignorant narrative that is being aggressively pushed despite lacking technical merit.

As things stand now, I object to this entire pull request.